mitodl/ocw-hugo-themes
# resources layout moved to base theme from courses

#### Pre-Flight checklist


#### What are the relevant tickets?
- Closes https://github.com/mitodl/ocw-hugo-themes/issues/708

#### What's this PR do?
- Creates a new partial in base-theme named `resource_layout.html` and moves all the code of `courses/layouts/resources/single.html` to this partial.
- Uses this partial in `courses/layouts/resources/single.html` and creates a new `resources layout` in www (i.e: `www/layouts/resources/`) which again uses this partial. 
- Moves some partials, which are being used in both themes, courses, and www from `courses` theme to `base` theme.
- Some little CSS tweaks

#### How should this be manually tested?
- Add some resources to `ocw-www`, like: https://ocw-studio-rc.odl.mit.edu/sites/ocw-www/type/resource/edit/65120dad-ad2a-41d8-98ac-54d9339b4468/
- Publish it.
- Take a pull on your local content. i.e: `WWW_CONTENT_PATH` 
- build `ocw-www` locally. i.e: `npm run start:www`
- Verify that all the resources added in step 1 are being loaded/server at `http://localhost:3000/resources/resource-slug`.
- Along with this, please verify that all the resources of different courses are being loaded/served like before. There is no change in course resources.
- Any other use-case which needs to be checked/tested.

#### Screenshots 
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/93309234/171122384-7fcb3a78-cc5d-4c28-a9cd-7626e95db031.png">

<img width="969" alt="image" src="https://user-images.githubusercontent.com/93309234/171122470-330299c2-96f6-4f94-99de-20e554217e50.png">
